src/test/java/core/basesyntax/service/impl/ReportCreatorServiceImplTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.ReportCreatorService;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportCreatorServiceImplTest {
+    private static ReportCreatorService reportCreatorService;
+    private static final String DEFAULT_TITLE = "fruit,quantity";
+    private static final int TITLE_INDEX = 0;
+    private static Map<String, Integer> testMap;
+
+    @BeforeClass
+    public static void setUp() {
+        reportCreatorService = new ReportCreatorServiceImpl();
+        testMap = new HashMap<>();
+    }
+
+    @After
+    public void afterEach() {
+        testMap.clear();
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void createReport_inputIsNull_notOk() {
+        reportCreatorService.createReport(null);
+    }
+
+    @Test
+    public void createReport_defaultTitle() {
+        testMap.put("banana", 44);
+        testMap.put("apple", 50);
+        testMap.put("orange", 123);
```

Comment:
Initialize your 'testMap' in setUp method and clear AfterClass, as you use it in two methods and each time do the same.